legend-engine-pure-code-compiled-core/src/main/resources/core/pure/router/preeval/preeval.pure

```diff
@@ -708,7 +708,33 @@ function <<access.private>> meta::pure::router::preeval::prevalGenericSimpleFunc
                                   modified = true
                                 );
                               }
-                            )                                                            
+                            ),
+                            pair(
+                              {theSfe:SimpleFunctionExpression[1] | ($theSfe.func == toOneMany_T_MANY__T_$1_MANY$_) && 

```

Comment:
If this will be a hot path, might be better to move away from the match, and use instanceOf/cast.  This will avoid creating an array of matches on each execution of these functions.

Reply:
Preeval is generally expected to be used as part of Service registration, not at actual execution time.